client/src/App.vue

```diff
@@ -1072,6 +1071,10 @@
 									</a>
 									).
 								</li>
+								<li>
+									Make use of your extracted cards here. Once the cards change in MTGA, you have to upload the
+									file again to see the most recent version.
+								</li>
```

Comment:
I don't think this step is necessary

Reply:
Reading the complete box again, I fully agree.